lib/http.js

```diff
@@ -0,0 +1,31 @@
+'use babel'
+
+import token from './token'
+
+function httpRequest(method, path, body, learnAuth=false) {
```

Comment:
at some point i'd like to reconcile our JS http stuff into one library we use across the board, either a 3rd party one we use as a standard, or our own implementation / wrapper